pkg/resources/kafka/kafka.go

```diff
@@ -160,6 +160,7 @@ func (r *Reconciler) Reconcile(log logr.Logger) error {
 	podList := &corev1.PodList{}
 	matchingLabels := client.MatchingLabels{
 		"kafka_cr": r.KafkaCluster.Name,
+		"app":      "kafka",
```

Comment:
I think rather `labelsForKafka(r.KafkaCluster.Name)` should used here as `labelsForKafka(r.KafkaCluster.Name)` is used when the Pod is created: https://github.com/banzaicloud/kafka-operator/blob/master/pkg/resources/kafka/pod.go#L88

Reply:
yup - better, fixed!